py3status/modules/xsel.py

```diff
@@ -24,25 +24,23 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 0.5
     command = 'xsel -o'
+    format = '{output}'
     max_size = 15
     symmetric = True
 
     def xsel(self):
-        """
-        Display the content of xsel.
-        """
         output = self.py3.command_output(self.command)
+        output = output.splitlines()[0]
```

Comment:
This is a change of behavior.  I understand the reason you've done it but I think we can do better.

options would be:

remove any `\n`
shown new line `\n` (bit ugly)
replace `\n` with ` ` (space)
replace with unicode special char eg `␤` or `↵`

how about things like `\t`?